resources/views/auth/login.blade.php

```diff
@@ -0,0 +1,89 @@
+    <link rel="stylesheet" href="{{ asset('fonts/material-icon/css/material-design-iconic-font.min.css')  }}">
+    <!-- Main css -->
+    <link rel="stylesheet" href="{{ asset('css/login.css') }}">
+    <link rel="stylesheet" href="{{ asset('css/app.css') }}">
+    <link rel="stylesheet" href="{{ asset('bower_components/flag-icon-css/css/flag-icon.css') }}">
+</head>
+
+<body>
+    <div class="main">
+        <!-- Sing in  Form -->
+        <section class="sign-in">
+            <div class="container">
+                <div class="signin-content">
+                    <div class="signin-image">
+                        <figure><img src="{{ asset('images/signin-image.jpg') }}" alt="sing up image"></figure>
+                        <a href="{{ route('register') }}" class="signup-image-link">{{ trans('new_account') }}</a>
+                    </div>
+                    <div class="signin-form">
+                        <h2 class="form-title">{{ trans('Login') }}</h2>
+                        <p class="require-field">* {{ trans('require_text')}}</p>
+                        <form method="POST" class="register-form" id="login-form" action="{{ route('login') }}">
+                            @csrf
+                            <div class="form-group">
+                                <label for="your_name"><i class="zmdi zmdi-account material-icons-name"></i></label>
+                                <input type="email" name="email" id="your_name"
+                                    class=" @error('email') is-invalid @enderror" placeholder="Your Email" />
+
+                                @error('email')
+                                <span class="invalid-feedback" role="alert">
+                                    <strong>{{ $message }}</strong>
+                                </span>
+                                @enderror
+                            </div>
+                            <div class="form-group">
+                                <label for="your_pass"><i class="zmdi zmdi-lock"></i></label>
+                                <input type="password" name="password" class="@error('password') is-invalid @enderror"
+                                    id="your_pass" placeholder="Password" />
+
+                                @error('password')
+                                <span class="invalid-feedback" role="alert">
+                                    <strong>{{ $message }}</strong>
+                                </span>
+                                @enderror
+                            </div>
+                            <div class="form-group">
+                                <input type="checkbox" name="remember" id="remember" class="form-check-input"
+                                    {{ old('remember') ? 'checked' : '' }} />
+                                <label for="remember-me" class="label-agree-term"><span><span></span></span>
+                                {{ trans('Remember me')}}</label>
```

Comment:
```suggestion
                                {{ trans('remember_me')}}</label>
```
mấy cái này nên đặt là key nhé, ko dùng text đâu